README.md

```diff
@@ -424,3 +424,17 @@ The example below presents how data is stored in Redux:
 ```
 
 The pageData entry is created on page load and destroyed when we close the tab. If the page does not contain Knot.x fragments, fragments property is empty.
+
+## Release
```

Comment:
what about version in manifest.json ? After yarn release command, version in manifest is 'increment'. Should we add this info to readme?